docs/src/pages/configurations/custom-webpack-config/index.md

```diff
@@ -5,9 +5,171 @@ title: 'Custom Webpack Config'
 
 You can customize Storybook's webpack setup by providing a `webpack.config.js` file exporting a **webpack 4** compatible config exported as a **commonjs module**.
 
-The file can export an object or a function. The former configures in [extend mode](#extend-mode), the latter in [full control mode](#full-control-mode), both described here.
+Storybook has its own Webpack setup and a dev server. 
+The webpack config [is configurable](/configurations/webpack), and the default can depend on which framework you're using and whether you've used a generator like [Create React App](https://github.com/facebookincubator/create-react-app) or Angular CLI etc.
 
-## Extend Mode
+> We're trying to make storybook more zero-config over time, **help to hook into the config of generators is very welcome**.
+
+<details>
+  <summary>This is what the config for storybook looks like when using CRA in dev-mode:</summary>
```

Comment:
Why do we put this in our docs? Can we just link to a file that won't get out of date every time somebody makes an change?

Reply:
There is no single file that has this info, it's all merged from multiple presets.

I think it's valuable for people who want to customize to quickly see what we're shipping with to most users.